picard/ui/infodialog.py

```diff
@@ -357,7 +357,7 @@ def _display_info_tab(self):
         tab_index = tabWidget.indexOf(tab)
         if album.errors:
             tabWidget.setTabText(tab_index, _("&Errors"))
-            text = '<br />'.join(map(lambda s: '<font color="darkred">%s</font>' %
```

Comment:
Does this look good on  light theme? We probably need to distinguish here. Same is actually true for the log window, maybe this could be fixed similar.

Dark layout can be checked with:

```
from picard.ui.theme import theme

theme.is_dark_theme
```
That's also what we use for switching syntax highlighting colors.

Reply:
What about making it configurable? Using https://github.com/metabrainz/picard/blob/master/picard/ui/colors.py

Reply:
Actually, I was just  thinking that. (I replaced a hard coded colour with a different hard coded colour.)

Leave this with me.

P.S. We could also do with an option to select system, light or dark theme.

Reply:
I am not going to fix any theme issues in this PR - just change the text colour.